src/go/configgenerator/filterconfig/filter_generator.go

```diff
@@ -266,6 +275,19 @@ func updateProtoDescriptor(service *confpb.Service, apiNames []string, descripto
 	return newData, nil
 }
 
+func preserveDefaultHttpBinding(httpRule *ahpb.HttpRule, defaultPath string) {
+	defaultBindingExisted := false
+	defaultBinding := &ahpb.HttpRule{Pattern: &ahpb.HttpRule_Post{defaultPath}, Body: "*"}
+	for _, addtionalBinding := range httpRule.AdditionalBindings {
+		if addtionalBinding == defaultBinding {
+			defaultBindingExisted = true
```

Comment:
This is address comparison not the deep comparison ?

Maybe add unit tests:
- with primary binding of default HTTP Pattern
- with additional bindings of default HTTP Pattern

Reply:
Can you clarify the first case - without httprule?

Reply:
If no httpRule annotation, no need to add the default HTTP Pattern as the transcoder will use it  automatically.

Reply:
The no httpRule annotation test case is added in UpdateProtoDescriptorTest.